client/checkout/blocks/upe.js

```diff
@@ -84,7 +84,22 @@ Object.entries( enabledPaymentMethodsConfig ).map( ( [ upeName, upeConfig ] ) =>
 		savedTokenComponent: <SavedTokenHandler api={ api } />,
 		canMakePayment: () => !! api.getStripe(),
 		paymentMethodId: upeMethods[ upeName ],
-		label: upeConfig.title,
+		// <span> element is an inline element which doesn't take the whole width
+		// of the parent component by default. This is why we need to `width: 95%` -
+		// so that the image can be aligned to the right side of the line with the help
+		// of the `float: right` property.
+		label: (
+			<>
+				<span style={ { width: '95%' } }>
+					{ upeConfig.title }
+					<img
+						style={ { float: 'right' } }
```

Comment:
Could we move these styles to `client/checkout/blocks/style.scss` rather than having them inline? Something like this: 

```
.wc-block-checkout__payment-method .wc-block-components-radio-control__label span {
	width: 95%;

	img {
		float: right;
	}
}
```

Reply:
Awesome, that's a great idea, thank you! Done in 1f919abf9620684e6773e160cbf81e0efea3cff6.